### PR TITLE
Initial drop of MPTelemetry 2.0 logging fat

### PR DESCRIPTION
**Pull request by tonyreigns**

### CODE REVIEW

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+WLP_LOGGING_CONSOLE_LOGLEVEL=off
+OTEL_LOGS_EXPORTER=logging
+OTEL_SDK_DISABLED=false
+COM_IBM_WS_LOGGING_CONSOLE_LOG.LEVEL=OFF
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.env]: Shouldn't we only need: `WLP_LOGGING_CONSOLE_LOGLEVEL=off`?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/bootstrap.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.console.log.level=OFF
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/bootstrap.properties]: don't think this is necessary?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <featureManager>
+	   <feature>pages-3.1</feature>
+       <feature>mpTelemetry-2.0</feature>
+       <feature>componentTest-2.0</feature>
+    </featureManager>
+	
+	<mpTelemetry source="ffdc"/>
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml]: indentation

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing the maxFfdcAge configuration attribute">
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <featureManager>
+	   <feature>pages-3.1</feature>
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml]: we need the pages feature?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryMessageNoApp/bootstrap.properties [hunk 5]
```diff
@@ -0,0 +1,2 @@
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.console.log.level=OFF
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryMessageNoApp/bootstrap.properties]: not needed

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryMessageNoApp/server.env [hunk 6]
```diff
@@ -0,0 +1,4 @@
+WLP_LOGGING_CONSOLE_LOGLEVEL=off
+OTEL_LOGS_EXPORTER=logging
+OTEL_SDK_DISABLED=false
+COM_IBM_WS_LOGGING_CONSOLE_LOG.LEVEL=OFF
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryMessageNoApp/server.env]: Shouldn't we only need: WLP_LOGGING_CONSOLE_LOGLEVEL=off?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryMessageNoApp/server.xml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<server description="system service">
+<logging traceSpecification="io.openliberty.microprofile.telemetry.internal.common=all:*=fine" />
+
+    <featureManager>
+       <feature>mpTelemetry-2.0</feature>
+       <feature>componentTest-2.0</feature>
+    </featureManager>
+
+     <include location="../fatTestPorts.xml"/>
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryMessageNoApp/server.xml]: indentation

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryTraceNoApp/bootstrap.properties [hunk 8]
```diff
@@ -0,0 +1,2 @@
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.console.log.level=OFF
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryTraceNoApp/bootstrap.properties]: not needed

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryTraceNoApp/server.env [hunk 9]
```diff
@@ -0,0 +1,4 @@
+WLP_LOGGING_CONSOLE_LOGLEVEL=off
+OTEL_LOGS_EXPORTER=logging
+OTEL_SDK_DISABLED=false
+COM_IBM_WS_LOGGING_CONSOLE_LOG.LEVEL=OFF
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryTraceNoApp/server.env]: Shouldn't we only need: WLP_LOGGING_CONSOLE_LOGLEVEL=off?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryTraceNoApp/server.xml [hunk 10]
```diff
@@ -0,0 +1,14 @@
+<server description="system service">
+<logging traceSpecification="io.openliberty.microprofile.telemetry.internal.common=all:*=fine" />
+
+    <featureManager>
+       <feature>mpTelemetry-2.0</feature>
+       <feature>componentTest-2.0</feature>
+    </featureManager>
+
+     <include location="../fatTestPorts.xml"/>
+
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryTraceNoApp/server.xml]: extra new lines

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/test-applications/ffdc-servlet/src/io/openliberty/microprofile/telemetry/logging/internal/fat/ffdc/servlet/FFDCServlet.java [hunk 11]
```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2018 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/test-applications/ffdc-servlet/src/io/openliberty/microprofile/telemetry/logging/internal/fat/ffdc/servlet/FFDCServlet.java]: change to only 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml [hunk 12]
```diff
@@ -0,0 +1,25 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml]: change to 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/bootstrap.properties [hunk 13]
```diff
@@ -0,0 +1,14 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/bootstrap.properties]: change to 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceTest.java [hunk 14]
```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceTest.java]: only need to mention 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceTest.java [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+        assertEquals("Random assert", 10, 10);
+
+    }
+
+    @Test
+    public void testTelemetryTrace() throws Exception {
+        // server.waitForStringInLog("CWWKE0002I", 10000);
+
+        String line = server.waitForStringInLog("server start complete.", server.getConsoleLogFile());
+
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceTest.java:46]: Need to asset on  log level and the actual message itself, it its present

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceTest.java [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+    @BeforeClass
+    public static void initialSetup() throws Exception {
+        server = LibertyServerFactory.getLibertyServer(SERVER_NAME);
+        server.startServer();
+    }
+
+    @Test
+    public void testMessageLog() throws Exception {
+        assertEquals("Random assert", 10, 10);
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceTest.java]: is this needed?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesTest.java [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    @Test
+    public void testTelemetryMessages() throws Exception {
+        String line = server.waitForStringInLog("CWWKF0011I", server.getConsoleLogFile());
+        List<String> linesMessagesLog = server.findStringsInFileInLibertyServerRoot("^(?!.*scopeInfo).*\\[.*$", MESSAGE_LOG);
+        List<String> linesConsoleLog = server.findStringsInFileInLibertyServerRoot(".*scopeInfo.*", CONSOLE_LOG);
+
+        assertEquals("Messages.log and Telemetry console logs don't match.", linesMessagesLog.size(), linesConsoleLog.size());
+
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesTest.java:58]: Need to asset on log level and the actual message itself, it its present

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java [hunk 18]
```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+ * Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java]: only 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesTest.java [hunk 19]
```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2023, 2024 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesTest.java]: only 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java [hunk 20]
```diff
@@ -0,0 +1,113 @@
...
+            // Restore the original server configuration, before starting the server for each test case.
+            server.restoreServerConfiguration();
+            server.startServer();
+        }
+    }
+
+    @Test
+    @ExpectedFFDC("java.lang.ArithmeticException")
+    public void testTelemetryMessages() throws Exception {
+        // server.waitForStringInLog("CWWKE0002I", 10000);
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCTest.java]: need this?

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/FATSuite.java [hunk 21]
```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2024 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/FATSuite.java]: only 2024

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/build.gradle [hunk 22]
```diff
@@ -0,0 +1,12 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2023 IBM Corporation and others.
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/build.gradle]: only 2024

**Review by pgunapal — CHANGES_REQUESTED** [review]
looks good, minor comments

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml (hunk 4 above)
Comment (tonyreigns) [reply dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/publish/servers/TelemetryFFDC/server.xml]: Yeah, for the FFDC servlet.

**Pull request closed without merging**